### ISSUE TITLE
Languages String does not return proper lang

### ISSUE THREAD

**Issue by cedricdcc** [vlizBE/vocabserver-app#70]
When testing the webcomponent it was found for this html config :

```html
<!DOCTYPE html>
<html>
<head>
        <title>Test vocab webcomponent</title>
        <script type="module" src="https://vocab.vliz.be/webcomponent/main.js"></script>
</head>

<body>
        <!-- The widget will be inserted here -->
        <div>
                <h1>Search</h1>
                <vocab-search-bar
                  search-endpoint="https://vocab.vliz.be/"
                  languages-string="nl"
                  >
                </vocab-search-bar>
               </body>
</html> 
```

The vocab that is loaded in are file dumps from DAMS.

* https://raw.githubusercontent.com/vlizBE/vocab-term-registry/main/dams/coordinate-reference-systems.ttl

It was found that this returned the concepts and showed them in english and not in the requested dutch.

**Comment by cedricdcc** [comment]
Upon further inspection I found that the language string does filter on the lang tag , so when given nl the results returned only contained concept which had a nl string. It is weird though that the english version was still shown in the search field though instead of the dutch version of the concept.
![image](https://github.com/user-attachments/assets/6ac19f89-ecfd-4c96-a801-756cbc72e355)

**Comment by MikiDi** [comment]
Hi Cedric,
As you show here, the `languages-string`-attribute is currently only used as an input to the backend search operation.  
The mode of display is independent from the provided `languages-string` and will always show all languages in the filter results box, and the first language result in the selected box.  
Just to clarify what we're aiming for: Both in the filter-results box as in the select-box, you'd like to show only the label in the language corresponding to the provided `languages-string`. For the above example that would be:
selected box: `Zuidpool stereo ...`
filter results box: `Zuidpool stereografisch`
Is that correct?

**Comment by cedricdcc** [comment]
Yes indeed so for this example instead of South Pole Stereographic it should have been zuidpool stereografisch

**Comment by nvdk** [comment]
if this is resolved, can the issue be closed?

**Comment by cedricdcc** [comment]
@MikiDi to be very clear here , do you guys still run the stack in development mode?
